includes/class-wc-payments-utils.php

```diff
@@ -910,6 +910,45 @@ public static function get_last_refund_from_order_id( $order_id ) {
 		return null;
 	}
 
+	/**
+	 * Get the name for the Core Payments task onboarding flow experiment.
+	 *
+	 * @return string
+	 */
+	public static function get_core_payments_task_onboarding_flow_abtest_name(): string {
+		return 'woopayments_core_payments_task_onboarding_flow_2024_v1';
+	}
+
+	/**
+	 * Get the shortname for the Core Payments task onboarding flow experiment.
+	 *
+	 * This will be used in our Tracks tracking, for brevity.
+	 *
+	 * @return string
+	 */
+	public static function get_core_payments_task_onboarding_flow_abtest_shortname(): string {
+		return 'cptof_2024_v1';
+	}
+
+	/**
+	 * Check to see if the current user is in Core Payments task onboarding flow experiment treatment mode.
+	 *
+	 * @return bool
+	 */
+	public static function is_in_core_payments_task_onboarding_flow_treatment_mode(): bool {
+		if ( ! isset( $_COOKIE['tk_ai'] ) ) {
+			return false;
+		}
+
+		$abtest = new \WCPay\Experimental_Abtest(
```

Comment:
Do we need to sanitized the `tk_ai` here or use it directly like [this PR](https://github.com/Automattic/woocommerce-payments/pull/2253) ? 

```new \WCPay\Experimental_Abtest(
	$_COOKIE[ 'tk_ai' ],
	'woocommerce',
	'yes' === get_option( 'woocommerce_allow_tracking' )
);```

Reply:
I followed the path used in [this recent experiment](https://github.com/Automattic/woocommerce-payments/pull/6376/files#diff-fdf97bb0bd6043ee97d2ab4f058059c59d2b7173a076398fe8441824dc7b5e35) we ran. Since the cookie value is just an ID, it can't hurt to sanitize it. On the other hand, the Abacus systems will surely sanitize it themselves. So, I don't know. I am kinda split here.

Reply:
should be fine, it looks like a additional check personally I'd way that `\WCPay\Experimental_Abtest` should have that baked into it!. Either way I was just checking it will not be a problem at all